packages/axios-extension/src/abap/ui5-abap-repository-service.ts

```diff
@@ -206,9 +213,12 @@ export class Ui5AbapRepositoryService extends ODataService {
                     ? '?sap-client=' + this.defaults.params['sap-client']
                     : '';
                 if (this.isDest) {
-                    this.log.info(
-                        '(Note: You will need to replace the host in the URL with the internal host, if your destination is configured using an On-Premise SAP Cloud Connector)'
-                    );
+                    const destinations = await listDestinations();
+                    if (isOnPremiseSystem(destinations[target.destination])) {
+                        this.log.info(
+                            '(Note: You will need to replace the host in the URL with the internal host, if your destination is configured using an On-Premise SAP Cloud Connector)'
```

Comment:
This error message doesnt make sense as we know its an onpremise destinaton at this stage. Might need to rever to PO to clarify this.